proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -254,3 +255,26 @@ message AuxSignerData {
   // sig is the signature of the sign doc.
   bytes sig = 4;
 }
+
+// AminoSignFee is the legacy amino json sign mode compatible version of txv1beta1.Fee, and differs from that message
+// by the name of the Gas field (GasLimit in txv1beta.Fee).
+// Note: this is only used for signing, see x/tx/signing/aminojson.go for more details.
```

Comment:
I'm hesitant to put this message in this package, which is a public one that's seens by a lot of people.

For textual, we had the same problem. We decided to put all textual-only proto types inside `x/tx/textual/internal/textualpb`, somewhat similar to what we do for local testpb files.

Or alternatively, inside the cosmos.tx.signing.v1beta1 package.

Reply:
That makes sense to me.  I think I'll follow the pattern in textual.